qa/L0_storage_azure/test.sh

```diff
@@ -179,20 +185,16 @@ if [ "$SERVER_PID" == "0" ]; then
 fi
 
 set +e
-for BACKEND in $BACKENDS; do
-    code=`curl -s -w %{http_code} -X POST localhost:8000/v2/repository/models/${BACKEND}_float32_float32_float32/load`
+for model in `ls models/`; do
```

Comment:
Why is this change needed?

Reply:
I'm now loading everything in the model dir, not just the float32 models, in order to run the unit tests which also test the `*_object` versions of the models.